specification/translation/Azure.AI.TextTranslation/models-languages.tsp

```diff
@@ -1,15 +1,17 @@
 import "@typespec/rest";
 import "@azure-tools/typespec-azure-core";
 import "./models-shared.tsp";
+import "@azure-tools/typespec-client-generator-core";
```

Comment:
Same comment as above for all the places where this import is used. You could move all of them to the same client.tsp file.

Reply:
Done